server/src/main/java/com/lunchtime/models/Restaurant.java

```diff
@@ -54,8 +51,10 @@
     @Column(name = "menu_id")
     private Long menuId;
 
-    @Column(name = "owner_id")
-    private Long ownerId;
+    @ManyToOne
+    @NotNull
+    @JoinColumn(name = "owner_id", nullable = false)
```

Comment:
We planned just with a little mistake our database. `owner_id` should be `user_id`.

Reply:
Fixed, but as we use `Person` entity (not `User` - Spring creates `user` table by default for system user) I changed column name to `person_id`

Reply:
Sure :)